DQ - Surviving Record/README.md

```diff
@@ -2,36 +2,46 @@
-* ### Surviving Options tab ###
+* ### Surviving Record ###
 
    | Standalone mode | Flow mode |
    | --- | --- |                  
    | ![](img/dqsurviving-tabsurvivingrecord-standalone.png) | ![](img/dqsurviving-tabsurvivingrecord-flowmode.png) |
-
-* Surviving Options
-1. **Sort No Dup**     - Performs a noduplicated sort on the entire row, it is mandatory to specify the Cluster_ID column and the by variables.  
-2. **Sort No Dup key** - Performs a noduplicated sort on the by variables, it is mandatory to specify the Cluster_ID column and the by variables.  
-3. **First record in cluster after sorting**   - Performs a sort on the Cluster_Id variable and select the first row of the cluster it is mandatory to specify the Cluster_ID.  
-4. **Last record in cluster after sorting**    - Performs a sort on the Cluster_Id variable and select the last row of the cluster it is mandatory to specify the Cluster_ID.  
-5. **Merge cluster using field rules** Allows you to define up to 5 rules on columns to select the best data within the cluster. Each rule is associated with a column, but it is also possible to specify more data as follows:  
-![DQ - Surviving Record Rule Sample](img/dqsurviving-SurvivingRecord_Rule.png)
-6. **Merge cluster using record rules** - Allows you to define up to 5 rules on columns to select the best data within the cluster. Each rule is associated with a column and will be applied to the entire record with the concept of record weight: the record with the greatest weight, that is, the one that has more data that satisfies the rule, will be kept.  
   
-* **Cluster ID** Defines the column that contains the cluster id required for all functions.  
-* **By columns** Defines the columns to sort the table used in the Sort Nodup and First and Last Cluster options.  
-* **Rule group** Section of the dialog contains rules that are used to determine which record in the cluster should be choosen as the surviving record. The section includes a pair of information (maximum 5), column and rule, to define the survival rule.  
-
-* ### Options tab ###  
-
-   ![](img/dqsurviving-tabsurvivingoptions-flowmode.png)  
+* ### Row Rules tab ###  
+Specifies the row-level rule that identifies which row in the cluster should be chosen as the surviving record.  
+
+1. **Select Cluster Column** - Specifies the cluster identifier column.  
+2. **Row Rule 1 - Select Column for row rule 1** - Specifies the input column to be used in the condition.
+3. **Select Operator** - Specifies the operator type. Valid operators include: missing, not missing, highocc (high occurrence), lowocc (low occurrence), longest, shortest, max, min, equal to, greater than, less than, greater or equal to, less than or equal to
+4. **Rule Operator** - Specifies the logical Boolean operator value to combine or exclude keywords in a search. Values can be AND or OR.
+5. **Row Rule 2** - Specifies the row rule parameters as shown in Row Rule 1.  
+  
+* ### Column Rules tab ###  
```

Comment:
Underneath this header I  would add a generic explanation of what column (level) rules are all about. Have taken the following (and adapted it slightly) from the [(good old DataFlux doc)](https://support.sas.com/documentation/onlinedoc/dfdmstudio/2.9/dfUnity.html#dfU_PFInt_SurvRecdID.html?Highlight=surviving).

_These rules enable you to substitute values into specific fields of the chosen surviving record._

Reply:
Have made this change myself in the README.md and updated the pull request